### PR TITLE
Create Raymond arizona recommendation file

### PR DESCRIPTION
**Pull request by RaymondSHANG**
- This is a bullet point.

- [ ] This is a task list item.

### CODE REVIEW

**Comment by bobby-gosselin** [comment]
General Comment

File: Raymond-Arizona.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Arizona
```
Comment (bobby-gosselin) [inline Raymond-Arizona.md:1]: ???

:+1:

Comment (RaymondSHANG) [reply Raymond-Arizona.md:1]: Thanks, you got it.

**Review by bobby-gosselin — APPROVED** [review]
Wooh! Merge

**Pull request merged**